README.md

```diff
@@ -2,55 +2,158 @@
 
 ![Dolphin Guy](http://i.giphy.com/RdkwfjGlsdnRm.gif)
 
-## Learning Objectives - The student should be able to..
+## Objectives
 
-* Explain that the `Bool` is a data type where the value can only be **true** or **false**.
-* Create variables using the Boolean literal values, `true` or `false` like so:
+1. Describe the `Bool` data type
+1. Create Boolean variables with literal values
+1. Use comparison operators on Boolean variables
+1. Use logic operators on Boolean variables
+
+## Understanding Booleans
+
+You are already familiar with creating `Int` variables and constants, which can store single numeric integer values, like
+
+```swift
+var numberOfApples = 7
+let numberOfOranges = 12
+```
+
+In this lesson, we will learn about the boolean data type, which represents a value that can either be true or false. Swift provides constant literal values for both possibilities, which can be used like the integer literals you are already familiar with:
+
+```swift
+let isTheSkyBlue = true
+let canHumansFly = false
+```
+
+Creating and using a variable of this type also works like the types you already know:
+
+```swift
+var isTheGrassGreen = false
+isTheGrassGreen = true
+```
+
+In both of these examples, the Swift compiler has inferred the type from our use of the literal values, but we can also explicitly write out the type if we want to:
+
+```swift
+let canDolphinsWalk: Bool = false
+```
+
+## Comparing values
+
+A common use of the `Bool` type is as a result of a comparison of other values. Comparing two values in Swift is done with an *operator*, which is a special symbol that you use to check, combine or change one or more values.
+
+For example, we can compare the two integer variables from our earlier example, to see if they are equal, using the equal-to operator:
+
```

Comment:
I think we should cover the equal-to operator in more detail here. This will be the first time they will be seeing `==`.

Stating something like:
Our `numberOfApples` variable above is of type `Int` with a value of 7. The `numberOfOranges` variable of type `Int` with a value of 12. In code, what if we wanted to pose the following question? "Is numberOfApples equal to numberOfOranges"? We do so using the equal-to operator which is represented by two equal signs `==`. Not to be confused with one equal sign `=` which is considered the assignment operator. So.. how do we structure that sentence in code:
```swift
numberOfApples == numberOfOranges
```
The `==` is a comparison of type `Bool`. In this case, we are asking "is 7 equal to 12" which is `false`. Here we aren't storing this comparison to a separate variable, which is something you will be doing.
```swift
let doWeHaveAsManyApplesAsOranges = numberOfApples == numberOfOranges
```
Here we are declaring a variable named `doWeHaveAsManyApplesAsOranges` which is followed by the `=` sign (assignment operator). Then we are met with a comparison using the `==` operator, this evaluates to `false` which becomes the value of the `doWeHaveAsManyApplesAsOranges` variable which is of type `Bool`.